release_notes/46-release-notes.adoc

```diff
@@ -0,0 +1,320 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="release-notes-45"]
+= Red{nbsp}Hat Advanced Cluster Security for Kubernetes 4.6
+include::modules/common-attributes.adoc[]
+:context: release-notes-46
+
+toc::[]
+
+{rh-rhacs-first} is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across the build, deploy, and runtime stages of the application lifecycle.
+{product-title} deploys into your infrastructure and integrates with your DevOps tools and workflows. This integration provides better security and compliance, enabling DevOps and InfoSec teams to operationalize security.
+
+.Release dates
+[options="header"]
+|====
+
+|{product-title-short} version |Released on
+
+|`4.6.0` | 2 December 2024
+
+|====
+
+[id="about-this-release-450_{context}"]
+== About release 4.6.0
+
+{product-title-short} 4.6 includes the following new features, improvements, and updates:
+
+[id="new-features_{context}"]
+== New features
+
+This release adds improvements related to the following components and concepts:
+
+[id="arm-support-secured-clusters_{context}"]
+=== Support for ARM architecture in secured clusters (Technology Preview)
+
+:FeatureName: Support for ARM architecture in secured clusters
+include::snippets/technology-preview.adoc[]
+
+We are adding support for ARM architecture in {product-title-short} secured clusters. This update allows users to leverage the efficient power consumption and high performance-per-watt benefits of ARM, making it ideal for resource-intensive tasks and cost-effective scaling and enhancing both flexibility and performance.
+
+For more information, see **LINK HERE**.
+
+[id="scanner-v4-updates_{context}"]
+=== Scanner V4 now uses CSAF/VEX for vulnerability data
+
+The vulnerability reporting for Red{nbsp}Hat content in {product-title-short} vulnerability management is changing with the introduction of Common Security Advisory Framework (CSAF) Vulnerability Exploitability eXchange (VEX) vulnerability data as a security data source. Red{nbsp}Hat is switching to CSAF VEX as a standardized security advisory format to communicate vulnerabilities affecting Red{nbsp}Hat products. In particular, the  VEX profile in CSAF format is now used to describe, with greater detail than before, which Red Hat products and components are impacted or known not to be impacted by a specific vulnerability identified by the Common Vulnerability and Exposures (CVE) ID.
+
+CSAF VEX is available when using Scanner V4 for vulnerability scanning.
```

Comment:
From the blog posts I linked above I think it's `CSAF-VEX` but honestly I have no clue. I'm just trying to use context clues but I could be wrong